src/main/java/no/unit/nva/cristin/projects/model/nva/NvaContributor.java

```diff
@@ -0,0 +1,44 @@
+package no.unit.nva.cristin.projects.model.nva;
+
+import static no.unit.nva.cristin.projects.JsonPropertyNames.AFFILIATION;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.IDENTITY;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.TYPE;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import nva.commons.core.JacocoGenerated;
+
+//import java.time.Instant;
+
+@JacocoGenerated
+@JsonPropertyOrder({TYPE, IDENTITY, AFFILIATION})
+public class NvaContributor {
+
+    private String type;
+    //private Instant startDate;
```

Comment:
Currently we cannot get these dates from backend API but it is a part of the NVA data model so i kept it for now